backend/services/esweService.ts

```diff
@@ -162,16 +164,74 @@ export default class EsweService {
     return createFlaggedContent(content)
   }
 
-  async getLearnerEmployabilitySkills(nomisId: string): Promise<LearnerEmployabilitySkills> {
-    let result: LearnerEmployabilitySkills = null
+  async getLearnerEmployabilitySkills(nomisId: string): Promise<EmployabilitySkills> {
+    const map = new Map<EmployabilitySkill, EmployabilitySkillsReview>()
     try {
       const context = await this.systemOauthClient.getClientCredentialsTokens()
-      result = await this.curiousApi.getLearnerEmployabilitySkills(context, nomisId)
+      const result = await this.curiousApi.getLearnerEmployabilitySkills(context, nomisId)
+
+      return createFlaggedContent(
+        result.content.reduce((acc, current) => {
```

Comment:
I think this `reduce` code block would benefit from been in it's own function with a name that indicates what it does.

One for later